style.css

```diff
@@ -26,20 +26,86 @@ body {
   align-items: center;
 }
 
-.logo p {
+.navbar .my-logo {
   color: var(--primary-b-400, #6070ff);
   font-size: 18px;
   font-weight: 700;
   line-height: 20px;
   text-align: center;
   margin-left: 12px;
+  margin-top: 12px;
   padding: 10px 12px;
+  text-decoration: none;
 }
 
-.menu-icon img {
-  margin-right: 24px;
-  padding: 8px;
-  vertical-align: middle;
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 30px;
+}
+
+@media screen and (max-width: 375px) {
+  .hamburger {
+    display: block;
+    cursor: pointer;
+    margin-right: 24px;
+    margin-top: 12px;
+  }
+
+  .bar {
+    display: block;
+    width: 25px;
+    height: 3px;
+    margin: 5px auto;
+    background: var(--primary-b-400, #6070ff);
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+    background-color: azure;
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+    background-color: azure;
+  }
+
+  .nav-menu {
+    position: fixed;
+    left: -100%;
+    top: 0;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 28px;
+    padding-top: 62px;
+    padding-bottom: 360px;
+    mix-blend-mode: multiply;
+    background: var(--theme-1-primary-colors-2, #6070ff);
+    width: 100%;
+    backdrop-filter: blur(0.4rem);
+    transition: 0.3s;
+  }
```

Comment:
- [ ] Great job on implementing the hamburger menu as specified in the project requirement! 👏 However, there is a minor UI issue 🐞 that we need to address and fix. Kindly consider making sure that your design looks as close to the `Figma` design as possible. 
  - Please make sure that your hamburger menu covers the full `height` of the main page as shown in the `Figma` design as this will help create a good user experience. See below for more details:
 <table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="373" alt="Screenshot 2023-06-20 at 15 09 17" src="https://github.com/78fatemanazari/Portfolio/assets/107192076/43493439-d06a-47aa-9ed1-f1535b2563cf">
</td>
<td>
<img width="373" alt="Screenshot 2023-06-26 at 20 42 47" src="https://github.com/78fatemanazari/Portfolio/assets/107192076/26ebbcbb-136a-4ad8-a1c4-25cd26781716">
</td>
</tr>
</table>